spec/oncocore_vs.txt

```diff
@@ -3,1316 +3,31 @@ Namespace:	oncocore
-ICD10CM#C8316	"Mantle cell lymphoma, intrapelvic lymph nodes"
-ICD10CM#C8317	"Mantle cell lymphoma, spleen"
-ICD10CM#C8318	"Mantle cell lymphoma, lymph nodes of multiple sites"
-ICD10CM#C8319	"Mantle cell lymphoma, extranodal and solid organ sites"
-ICD10CM#C8330	"Diffuse large B-cell lymphoma, unspecified site"
-ICD10CM#C8331	"Diffuse large B-cell lymphoma, lymph nodes of head, face, and neck"
-ICD10CM#C8332	"Diffuse large B-cell lymphoma, intrathoracic lymph nodes"
-ICD10CM#C8333	"Diffuse large B-cell lymphoma, intra-abdominal lymph nodes"
-ICD10CM#C8334	"Diffuse large B-cell lymphoma, lymph nodes of axilla and upper limb"
-ICD10CM#C8335	"Diffuse large B-cell lymphoma, lymph nodes of inguinal region and lower limb"
-ICD10CM#C8336	"Diffuse large B-cell lymphoma, intrapelvic lymph nodes"
-ICD10CM#C8337	"Diffuse large B-cell lymphoma, spleen"
-ICD10CM#C8338	"Diffuse large B-cell lymphoma, lymph nodes of multiple sites"
-ICD10CM#C8339	"Diffuse large B-cell lymphoma, extranodal and solid organ sites"
-ICD10CM#C8350	"Lymphoblastic (diffuse) lymphoma, unspecified site"
-ICD10CM#C8351	"Lymphoblastic (diffuse) lymphoma, lymph nodes of head, face, and neck"
-ICD10CM#C8352	"Lymphoblastic (diffuse) lymphoma, intrathoracic lymph nodes"
-ICD10CM#C8353	"Lymphoblastic (diffuse) lymphoma, intra-abdominal lymph nodes"
-ICD10CM#C8354	"Lymphoblastic (diffuse) lymphoma, lymph nodes of axilla and upper limb"
-ICD10CM#C8355	"Lymphoblastic (diffuse) lymphoma, lymph nodes of inguinal region and lower limb"
-ICD10CM#C8356	"Lymphoblastic (diffuse) lymphoma, intrapelvic lymph nodes"
-ICD10CM#C8357	"Lymphoblastic (diffuse) lymphoma, spleen"
-ICD10CM#C8358	"Lymphoblastic (diffuse) lymphoma, lymph nodes of multiple sites"
-ICD10CM#C8359	"Lymphoblastic (diffuse) lymphoma, extranodal and solid organ sites"
-ICD10CM#C8370	"Burkitt lymphoma, unspecified site"
-ICD10CM#C8371	"Burkitt lymphoma, lymph nodes of head, face, and neck"
-ICD10CM#C8372	"Burkitt lymphoma, intrathoracic lymph nodes"
-ICD10CM#C8373	"Burkitt lymphoma, intra-abdominal lymph nodes"
-ICD10CM#C8374	"Burkitt lymphoma, lymph nodes of axilla and upper limb"
-ICD10CM#C8375	"Burkitt lymphoma, lymph nodes of inguinal region and lower limb"
-ICD10CM#C8376	"Burkitt lymphoma, intrapelvic lymph nodes"
-ICD10CM#C8377	"Burkitt lymphoma, spleen"
-ICD10CM#C8378	"Burkitt lymphoma, lymph nodes of multiple sites"
-ICD10CM#C8379	"Burkitt lymphoma, extranodal and solid organ sites"
-ICD10CM#C8380	"Other non-follicular lymphoma, unspecified site"
-ICD10CM#C8381	"Other non-follicular lymphoma, lymph nodes of head, face, and neck"
-ICD10CM#C8382	"Other non-follicular lymphoma, intrathoracic lymph nodes"
-ICD10CM#C8383	"Other non-follicular lymphoma, intra-abdominal lymph nodes"
-ICD10CM#C8384	"Other non-follicular lymphoma, lymph nodes of axilla and upper limb"
-ICD10CM#C8385	"Other non-follicular lymphoma, lymph nodes of inguinal region and lower limb"
-ICD10CM#C8386	"Other non-follicular lymphoma, intrapelvic l
```

Comment:
Why don't we have ICD-10-CM codes with specified histologies and behavior here?

Reply:
I think we need to add behavior extensions, if that's what you mean. It will really blow up the size of the set.